news/create-newsfile.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+#############################################################################
+# Copyright (c) 2020 Balabit
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 as published
+# by the Free Software Foundation, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+#
+# As an additional exemption you are allowed to compile & link against the
+# OpenSSL libraries as published by the OpenSSL project. See the file
+# COPYING for details.
+#
+#############################################################################
+
+import re
+import sys
+from pathlib import Path
+from time import localtime, strftime
+
+
+script_dir = Path(__file__).parent
+root_dir = script_dir.parent
+newsfile = (root_dir / 'NEWS.md')
+
+
+def create_block(block_name, files):
+    block = '## {}\n\n'.format(block_name)
+    for f in files:
+        entry = ''
+        try:
+            pr_id = re.findall(r'\d+.md$', f.name)[0][:-3]
+        except IndexError:
+            sys.exit('Invalid filename: {}'.format(f.name))
+        entry += ' * {} (#{})\n'.format(f.read_text().rstrip(), pr_id)
+        entry = entry.replace('\n', '\n   ')
```

Comment:
When we replace "\n" with "\n   " it will broke the indentation of the second line of content, for example:
Reproduction:
- just create 2 separate "feature" file in news/ dir
```
## Features

 * bbbbbbbbbb (#222222)
    * ffffffffffffffffffff (#999999)
```

Reply:
This is intentional, this will create a sub-bulletpoint for you. Is there any reason to create 2 entries for one PR?

Reply:
```
aaaa
 * bbbb
```
which is:
aaaa
 * bbbb

will become:
* aaaa
   * bbbb

Reply:
Sorry, I misunderstood.
```
        entry += ' * {} (#{})'.format(f.read_text().rstrip(), pr_id)
        entry = entry.replace('\n', '\n   ') + '\n'
        block += entry
```
will fix the problem